package.json

```diff
@@ -44,6 +44,7 @@
   },
   "dependencies": {
     "babel-runtime": "^5.6.20",
+    "graphql-relay": "^0.3.2",
```

Comment:
Yeah i saw that, ideally i'd like to avoid that, if the user is using relay they'll add it and we should still be able to require it, we just need some way of detecting when to include relay stuff.